### PR TITLE
Prepare for core-kit 3.0.0 release

### PR DESCRIPTION
**Pull request by ytsssun**
**Description of changes:**
- Changelog for 3.0.0 release.
- Bump core-kit version in `Twoliter.toml`

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+# v3.0.0 (2024-10-17)
+
+## OS Changes
+* Drop kubernetes-1.23 ([#184])
+* Set Epoch to 1 in necessary packages ([#180])
```
Comment (bcressey) [inline CHANGELOG.md]: This can move to build changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+* Move kmod-5.10-nvidia to R535 branch from R470 ([#181])
+* Block writeable/executable memory in systemd services by default ([#158])
+* Drop dependency on glibc for nvidia kmods ([#194])
```
Comment (bcressey) [inline CHANGELOG.md]: This is also a build change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
+# v3.0.0 (2024-10-17)
+
+## OS Changes
+* Drop kubernetes-1.23 ([#184])
+* Set Epoch to 1 in necessary packages ([#180])
+* Move kmod-5.10-nvidia to R535 branch from R470 ([#181])
```
Comment (arnaldo2792) [inline CHANGELOG.md]:
```suggestion
* Move kmod-5.10-nvidia from branch R470 to R535 ([#181])
```

Comment (ytsssun) [reply CHANGELOG.md]: The current log is consistent with the original [PR](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/181/commits), so I would vote to keep it as it is. Or the commit owner @yeazelm can chime in.

Comment (yeazelm) [reply CHANGELOG.md]: I'm fine with the change. It reads more clearly.

**Review by vigh-m — APPROVED** [review]

**Review by cbgbt — APPROVED** [review]

**Review by cbgbt — APPROVED** [review]

**Pull request merged**